### PR TITLE
#P15. As Participant, I would like to see what timezone the members of my team are in.

### PR DESCRIPTION
**Pull request by stefdworschak**
## Description
- Adding a new field to capture a user's timezone
- Adding a view to display the difference in timezones

## Pull request type

- [ ] #M01 (Miscellaneous User Story #01)
- [x] #P02 (Participant User Story #02)
- [ ] #S03 (Staff User Story #03)
- [ ] #A04 (Admin User Story #04)
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Related Issue
#203

## Configuration instructions
- Run migrations

## Screenshots

Screenshot from 2021-12-02 14-38-05
Screenshot from 2021-12-02 14-38-09

### CODE REVIEW

File: custom_slack_provider/views.py [hunk 1]
```diff
@@ -76,15 +76,16 @@ def get_data(self, token):
         resp['user']['full_name'] = user_info.get('profile',
                                                   {}).get('real_name')
         resp['user']['first_name'] = user_info.get('profile',
-                                                  {}).get('first_name')
+                                                   {}).get('first_name')
```
Comment (TravelTimN) [inline custom_slack_provider/views.py:79]: Is this one indented properly? They rest look 1-space to the left still.

File: teams/templates/team_calendar.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+            <div class="row">
+                <div class="col">
+                    
+                    <table class="table table-responsive-sm table-sm calendar">
+                        <thead>
+                            <tr>
+                                {% for header in headers %}
+                                <td title="{{header.description}}">
+                                    <strong>{{header.display_name}}</strong><br />
```
Comment (TravelTimN) [inline teams/templates/team_calendar.html:64]: HTML doesn't require self-closing `</>` at the end.

File: teams/templatetags/teams_tags.py [hunk 3]
```diff
@@ -39,3 +39,10 @@ def divided_by(num, val):
 @register.filter
 def extract_userid(username):
     return username.split('_')[0]
+
+
+@register.filter
+def is_working_time(num):
+    _num = int(num.split(':')[0])
+    return 8 <= _num <= 20
+    return num
```
Comment (TravelTimN) [inline teams/templatetags/teams_tags.py:48]: Missing blank line at EOF

Comment (TravelTimN) [reply teams/templatetags/teams_tags.py:48]: Why 2x `return` statements at same indentation? Second one cannot be reached, no?

**Review by TravelTimN — APPROVED** [review]
LGTM! 👍🏻 
A few comments/questions/suggestions, but all approved otherwise.

File: teams/templates/team_calendar.html (hunk 2 above)
Comment (stefdworschak) [reply teams/templates/team_calendar.html]: Removed `</>` on  the self-closing tag.

File: teams/templatetags/teams_tags.py (hunk 3 above)
Comment (stefdworschak) [reply teams/templatetags/teams_tags.py]: Removed the second return and added blank line

File: custom_slack_provider/views.py (hunk 1 above)
Comment (stefdworschak) [reply custom_slack_provider/views.py:78]: I am pretty sure it's correct. It should be in line with `'profile'` which it is now.

**Pull request merged**